### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Psiale**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   </main>
+
+</div>
+
+   <section>
+<div class="flex-images-section-container">
+<img src="images/promo_iphone5s.jpg" alt="iphone image">
+<img src="images/promo_iphone5c.jpg" alt="iphone image">
+<img src="images/promo_30_years.jpg" alt="iphone image">
+<img src="images/promo_supplier_responsibility.jpg" alt="iphone image">
```
Comment (suretrust) [inline index.html]: These images should be wrapped in an `<a>` tag since they are links.

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  text-decoration: underline;
+}
+
+.flex-link-and-icon-container > .main-icon {
+  width: 30px;
+  height: 30px;
+}
+
+.flex-images-section-container {
+  width: 100vh;
```
Comment (suretrust) [inline styles/style.css]: This should be `vw` right? Kindly refer to [this article](http://www.tipue.com/blog/css3-vw-vh/) for clarity.

Also, the overall width of the images all together should align with the width of the background image above. 

width

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @Psiale,

### You have done a great job overall. :+1: 

However, your code seems scattered. Let's make it look cleaner. You can install `Prettier - Code Formatter` on your code editor to make your codebase _prettier_. :wink:  If you use VSCode, you can check [this article](https://scotch.io/tutorials/code-formatting-with-prettier-in-visual-studio-code#toc-installing-the-prettier-extension) to read the _how to_.

Also, kindly check my comments on your code.

**Please submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Review by ebukaume — CHANGES_REQUESTED** [review]
## Changes Required! :stop_sign: 

Hi @Psiale,

Please attend to the issue pointed out by @suretrust in the last review. The 4 images on the bottom part of the page have to be links. _Remember `<a>`?_

Submit another code review request after fixing this.

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @Psiale,

#### Great to see you have fixed some of the issues, :+1: however, the width is still wider than the background image and let's get rid of the horizontal navbar.

width

**Kindly submit for another code review once these issues have been fixed**. _Happy coding!_ :tada:

**Review by suretrust — APPROVED** [review]
### Amazing job! :+1: Let's get this merged! :tada: 

_Good luck with your future projects and happy coding!_

**Pull request merged**